File: snippets/api/getting-started/bungee.java
package com.plasmovoice.bungeeplugin;

import su.plo.voice.api.proxy.PlasmoVoiceProxy;

import net.md_5.bungee.api.plugin.Plugin;

public final class BungeePlugin extends Plugin {

    // Addon class annotated with @Addon.
    private final TestAddon addon = new TestAddon();

    @Override
    public void onLoad() {
        PlasmoVoiceProxy.getAddonsLoader().load(addon);
    }
}

File: snippets/api/getting-started/bungee.kt
package com.plasmovoice.bungeeplugin

import su.plo.voice.api.proxy.PlasmoVoiceProxy

import net.md_5.bungee.api.plugin.Plugin

class BungeePlugin : Plugin() {

    // Addon class annotated with @Addon.
    private val addon = TestAddon()

    override fun onLoad() {
        PlasmoVoiceProxy.getAddonsLoader().load(addon)
    }
}
